css/templates/e-commerce/page-vars.css

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2017 The AMP Start Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+:root {
```

Comment:
on your page in general i noticed  if you set your primary/seconday color as white and black and accent color as pink - you should be able to get most of the color theme right (without having to override a lot) Try this and see if it works ,  else we can proceed with what you have